docs/operating-eck/troubleshooting.asciidoc

```diff
@@ -7,22 +7,24 @@ endif::[]
 [id="{p}-{page_id}"]
 = Troubleshoot ECK
 
-When things don't work as expected, you can investigate by taking the following actions:
+Most common issues can be identified and resolved by following these instructions:
 
-- <<{p}-get-resources,Get the list of resources>>
+- <<{p}-get-resources,View the list of resources>>
 - <<{p}-describe-failing-resources,Describe failing resources>>
-- <<{p}-get-elasticsearch-logs,Get Elasticsearch logs>>
-- <<{p}-get-init-container-logs,Get init container logs>>
-- <<{p}-get-eck-logs,Get ECK logs>>
+- <<{p}-view-logs>>
 - <<{p}-eck-debug-logs,Enable ECK debug logs>>
```

Comment:
This section comes actually before "View logs"

Reply:
Good catch! I moved the section text but forgot to update the list.